sql/table.cc

```diff
@@ -7774,3 +7777,17 @@ void TABLE::set_pos_in_table_list(TABLE_LIST *table_list) noexcept {
 }
 
 //////////////////////////////////////////////////////////////////////////
+
+const time_point get_time_now() { return std::chrono::system_clock::now(); }
```

Comment:
IMHO this wrapper is not worth it, system_clock::now() can be just called directly.
But otherwise "static"